DBM-GUI/localization.br.lua

```diff
@@ -39,13 +39,13 @@ L.MoveMe 					= "Mova-me"
 L.Button_OK 				= "OK"
 L.Button_Cancel 			= "Cancelar"
 L.Button_LoadMod 			= "Carregar AddOn"
-L.Mod_Enabled				= "Habilitar: %s"
+L.Mod_Enabled				= "Ativar: %s"
 L.Mod_Reset					= "Carregar opções padrão"
-L.Reset 					= "Resetar"
```

Comment:
I would keep it Resetar. It is an anglicism, sure, but one that most brazillians I've played with use.

Reply:
Blizz uses Redefinir, although I agree both are usable I just tend to make everything follow blizz localization
![image](https://github.com/user-attachments/assets/f45fd36c-a49c-4b9a-bf0d-e05dd0e1a468)

Reply:
Agreed.